### DISCUSSION TITLE
Does it support  download into ALTERA MAX10?

### DISCUSSION THREAD

**Discussion by top58525785** [logisim-evolution/logisim-evolution#1842] [Q&A] [answered]
Hello
everybody
Does logsim  support download into  MAX10？I add FPGA model is 10M08SCM153

when i download  program into  fpga, It shows error as below:

Info: Command: quartus_map logisimTopLevelShell --optimize=area
Error (125095): Part name 10M08SC153 is invalid
    Info (125063): set_global_assignment -name DEVICE 10M08SC153
Error (281000): Part name 10M08SC153 is illegal

**Comment by maehne** [comment, marked as answer]
Yes, see your issue #1850 for the answer.
